forms-flow-web/src/components/PrivateRoute.jsx

```diff
@@ -237,6 +238,9 @@ const PrivateRoute = React.memo((props) => {
             {ENABLE_FORMS_MODULE && (
               <Route path={`${BASE_ROUTE}form`} component={Form} />
             )}
+            {ENABLE_FORMS_MODULE && (
+              <Route path={`${BASE_ROUTE}landing`} component={LandingPage} />
```

Comment:
this route is available for those who not logged in the application  also right could you please check